libs/net/kafka/Models/ContentBase.cs

```diff
@@ -119,6 +119,11 @@ public abstract class ContentBase
     /// </summary>
     public IEnumerable<Tag> Tags { get; set; } = Array.Empty<Tag>();
 
+    /// <summary>
+    /// get/set - An array of labels to identify this content.
+    /// </summary>
+    public IEnumerable<ContentLabel> Labels { get; set; } = Array.Empty<ContentLabel>();
```

Comment:
I think you're referencing an entity here.  It should be a model from the Kafka library

Reply:
no, it's referencing the Kafka model which is new.  I notice you had a TODO around changing the Kafka Model names be suffixed with Model wo when I created this one I took that step in advance
![image](https://github.com/bcgov/tno/assets/63015960/9cd9cdca-4d59-435e-a871-7ca8fe9f8116)